docs/README.md

```diff
@@ -99,16 +99,16 @@ environment variables as parameters:
 
 | Parameter      | Description                                                      | Required | Default value                                                                     |
 |----------------|------------------------------------------------------------------|----------|-----------------------------------------------------------------------------------|
-| `DISTRIBUTION` | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows` | Yes      |                                                                                   |
 | `INSTALL_DIR`  | Location where binaries are to be installed                      | No       | `./otel-dotnet-auto`                                                              |
+| `OS_TYPE`      | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows` | Yes      |                                                                                   |
```

Comment:
Why no default? Possibly a silly question. Alternatively, I'd add a sentence in that column stating how the value gets filled.

Reply:
Good question.

I thought about setting `linux-glibc` as default. However, when one will execute it on another OS (e.g. Windows), she/he will have a really hard time understanding why the instrumentation is not working.

Calculating the default value based on the machine info is hard and I do not want us to maintain this code.